projects/can_api/tests/rx_mask.yml

```diff
@@ -0,0 +1,5 @@
+name: rx_only
+subscribe:
+  - name: test_tx
+    mob: 0
```

Comment:
Nit: by our team's convention, MOb 0 is only used for TX. This is fine in theory, but for sake of continuity can you make it 1?

Reply:
ah yes, will do